lib/engine/game/g_1848/game.rb

```diff
@@ -530,9 +530,8 @@ def after_buy_company(player, company, _price)
         end
 
         def lookup_boe_price(p)
-          @stock_market.market[BOE_ROW].size.times do |i|
-            next unless @stock_market.share_price(BOE_ROW, i)
-            return @stock_market.share_price(BOE_ROW, i) if @stock_market.share_price(BOE_ROW, i).price == p
+          @stock_market.market[BOE_ROW].each do |i|
```

Comment:
It's best practice to be more descriptive in the block args to give context of what is being iterated. For example, here I'd used sp for share_price.

Reply:
Let me know if you want to make that change before I merge.

Reply:
done. can merge

Reply:
I was referring to the block variable - i, which represents an array of share prices. It makes it easier to maintain when the block variable gives context as to what it is. the method param was fine -- though price would have been more descriptive there.

Reply:
done